docs/pkg-set.8

```diff
@@ -49,38 +40,47 @@ should always be used with caution.
 The following options are supported by
 .Nm :
 .Bl -tag -width automatic
-.It Fl A Ar 01 , Cm --automatic Ar 01
-Set automatic flag for the package: 0 is not automatic, 1 is automatic.
+.It Fl A Ar 0|1 , Fl -automatic Ar 0|1
+Set the
+.Qq automatic
+flag for the package: 0 is not automatic, 1 is automatic.
 This affects the operation of
 .Xr pkg-autoremove 8 .
-.It Fl a , Cm --all
+.It Fl a , Fl -all
 Match all installed packages.
-.It Fl C , Cm --case-sensitive
-Make the standard or the regular expression
-.Fl ( x )
+This is the default unless
+.Ar pkg-name
+is present.
+.It Fl C , Fl -case-sensitive
+Make the globbing
+.Pq Fl g
+or regular expression
+.Pq Fl x
 matching against
 .Ar pkg-name
 case sensitive.
-.It Fl g , Cm --glob
+.It Fl g , Fl -glob
 Match
 .Ar pkg-name
 as a globbing expression.
-.It Fl i , Cm --case-insensitive
-Make the standard or regular expression
-.Fl ( x )
+.It Fl i , Fl -case-insensitive
+Make the globbing
+.Pq Fl g
+or regular expression
+.Pq Fl x
 matching against
 .Ar pkg-name
-case insensitive.
+case-insensitive.
 This is the default, unless modified by setting
 .Ev CASE_SENSITIVE_MATCH
 to true in
 .Pa pkg.conf .
```

Comment:
```suggestion
.Xr pkg.conf 5 .
```
I think xrefs take highest priority when there are multiple appropriate macros since they're clickable in most implementations.